code-build.js

```diff
@@ -296,8 +305,8 @@ function inputs2Parameters(inputs) {
   const sourceOverride = !disableSourceOverride
     ? {
         sourceVersion: sourceVersion,
-        sourceTypeOverride: "GITHUB",
-        sourceLocationOverride: `https://github.com/${owner}/${repo}.git`,
+        sourceTypeOverride: sourceTypeOverride || "GITHUB",
+        sourceLocationOverride: sourceLocationOverride || `https://github.com/${owner}/${repo}.git`,
```

Comment:
could you please add an unit test ? 

you can find an example here: https://github.com/aws-actions/aws-codebuild-run-build/blob/main/test/code-build-test.js#L410-L421